powerplant-android/src/main/java/org/kaaproject/kaa/examples/powerplant/PowerPlantAndroidDemoBuilder.java

```diff
@@ -56,12 +59,18 @@ protected void buildDemoApplicationImpl(AdminClient client) throws Exception {
 
         loginTenantDeveloper(client);
 
+        logger.info("Creating ctl schema...");
+
+        CTLSchemaDto ctlSchema = client.saveCTLSchemaWithAppToken(getResourceAsString("config_schema.avsc"), powerPlantAndroidApplciation.getTenantId(), powerPlantAndroidApplciation.getApplicationToken());
```

Comment:
Then you can remove "ctlSchema" local variable and use
saveCTLSchemaWithAppToken("config_schema.avsc", powerPlantAndroidApplciation).getId()
or even return id of the schema instead of CTLSchemaDto.